pkg/helpers/helpers.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"fmt"
+	"strings"
+)
+
+// CombineErrors combines the messages from multiple errors into a single error. It returns nil if all errors are nil.
+func CombineErrors(errs []error) error {
```

Comment:
minor nit: if you use a [variadic](https://go.dev/ref/spec#Passing_arguments_to_..._parameters) function then it will look a little nicer at the call site:

`err = helpers.CombineErrors(err, cerr)`

Reply:
Vari cool! <yuck yuck>

Reply:
Pun intended 😁